implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java

```diff
@@ -95,19 +89,24 @@ private DimensionList parseDefaultDimensions(Map<String, String> defaultDimensio
         return DimensionList.fromCollection(dimensions);
     }
 
+    /**
+     * Export to the Dynatrace v2 endpoint. Metrics that contain NaN or Infinite values, as well as
+     * lines that are too long after serialization will be dropped and not exported. If the number
+     * of lines exceeds the maximum number of lines per request, lines will be sent in chunks.
+     *
+     * @param meters A list of {@link Meter Meters} that are serialized as one or more metric lines.
+     */
     @Override
     public void export(@Nonnull List<Meter> meters) {
-        Map<Boolean, List<String>> metricLines =
+        // Lines that are too long to be ingested into Dynatrace, as well as lines that contain NaN
```

Comment:
Here it's fine to talk about lines since it's clear from the context (`List<String>`).